projects/core/src/user/store/reducers/transitional.ts

```diff
@@ -5,54 +5,32 @@ import { PaymentDetails } from '../../../model/cart.model';
 import { ConsentTemplate } from '../../../model/consent.model';
 import { CustomerCouponSearchResult } from '../../../model/customer-coupon.model';
 import { NotificationPreference } from '../../../model/notification-preference.model';
-import {
-  Order,
-  OrderHistoryList,
-  ReturnRequest,
-  ReturnRequestList,
-} from '../../../model/order.model';
```

Comment:
Aren't these removals breakiinf changes?

Reply:
It is not breaking changes. But to keep the old codes work, I will revert it back.